collect/central_collector_test.go

```diff
@@ -186,7 +186,7 @@ func TestCentralCollector_ProcessTraces(t *testing.T) {
 				assert.Equal(t, numberOfTraces*10, len(transmission.Events))
 				assert.Equal(t, "aoeu", transmission.Events[0].Dataset)
 				assert.Equal(t, "test", transmission.Events[0].Environment)
-				assert.Equal(t, TraceSendGotRoot, transmission.Events[0].Data["meta.refinery.send_reason"])
+				assert.Equal(t, TraceSendGotRoot, transmission.Events[0].Data["meta.refinery.send_reason"], transmission.Events[0])
```

Comment:
maybe this wants to be a format string with a message instead of  just an extra parameter?

Reply:
I didn't mean to commit this